airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresJdbcSourceAcceptanceTest.java

```diff
@@ -303,6 +303,10 @@ void testSpec() throws Exception {
 
   @Override
   protected List<AirbyteMessage> getTestMessages() {
```

Comment:
Seems like in the new `CtidEnabledPostgres` class you're just using this getTestMessages() method as a helper function? 

IMO you should just copy pasta the entire test (similar to what you had earlier) and have CtidEnabledPostgresSourceTest inheirit directly from JdbcSourceAcceptanceTest. That way, gutting it when we've made the transition becomes easy (just a matter of deleting the old file). WDYT?

Reply:
Honestly I'm more down for that. I try to keep things clean, but if we're planning to gut it, copy pasta is the better move.